### PR TITLE
Add tools for maintaining ChangeLog

### PR DESCRIPTION
**Pull request by morevnaproject**
This PR contains scripts to handle generating of ChangeLog. The trick is to get ChangeLog generated and avoid mixing of commits from **stable** and **master** branches.

### CODE REVIEW

**Comment by rodolforg** [comment]
@morevnaproject I don't know if it works everywhere, but https://markdownlivepreview.com/ renders fine this:
```- [`9700b67`](https://github.com/synfig/synfig/commit/9700b677051ed95660887938a95a784576502ba0) Export video files with sound [core]```

(Oh, here too ;) )

- [`9700b67`](https://github.com/synfig/synfig/commit/9700b677051ed95660887938a95a784576502ba0) Export video files with sound [core]

**Comment by morevnaproject** [comment]
Cool! Thank you!

**Comment by morevnaproject** [comment]
Okay, this is ready for merging! Someone, approve, please.  ^__^

File: ChangeLog-split.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+
+# https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself
+SOURCE="${BASH_SOURCE[0]}"
+DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
```
Comment (ice0) [inline ChangeLog-split.sh:6]: Why not just use?
```
dirname "$0"
```

Comment (morevnaproject) [reply ChangeLog-split.sh:6]: Please see link in comment just above - https://stackoverflow.com/questions/59895/how-can-i-get-the-source-directory-of-a-bash-script-from-within-the-script-itsel
In short: it resolves symlinks and gives real path. I am not sure if that matters for case of this script, just use this solution everywhere to be safe. ^__^

**Review by ice0 — APPROVED** [review]

**Pull request merged**